drivers/bus/acpi/eval.c

```diff
@@ -713,9 +687,7 @@ EvalEvaluateObject(
 {
     CHAR MethodName[ACPI_OBJECT_NAME_LENGTH];
 
-    PAGED_CODE();
-
-    RtlCopyMemory(MethodName, EvalInputBuffer->MethodName, ACPI_OBJECT_NAME_LENGTH);
+    RtlCopyMemory(MethodName, EvalInputBuffer->MethodName, ACPI_OBJECT_NAME_LENGTH - 1);
```

Comment:
The `EvalInputBuffer->MethodName` isn't NULL-terminated I suppose?

Reply:
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/ddk/acpiioct.h;hb=b6562a664bef4418d39dd5801dd2adc7d1d0f1f6#l38